Source/EasyGelf.Log4Net/GelfTcpAppender.cs

```diff
@@ -10,7 +8,7 @@ public sealed class GelfTcpAppender : GelfAppenderBase
     {
         public GelfTcpAppender()
         {
-            TcpTransportConfiguration defaultCfg = TcpTransportConfiguration.GetDefaultConfiguration();
+            var defaultCfg = TcpTransportConfiguration.GetDefaultConfiguration();
```

Comment:
Eh, its not worth the bloat and loss of readability to me. Any modern text editor or IDE can show you the compiler inferred type with a simple mouse hover. Its also rather difficult for me to memorize the particulars of a type or contract when working across many projects, so I regularly rely on my tools to provide that resolution when needed.